models/topicsModel.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../db/connection");
+
+const fetchAllTopics = (sort_by = "slug") => {
+  // this is fully SQL-injection proof!
+  const validSortQueries = ["slug", "description"];
+  // const validOrderQueries = ["asc", "desc"];
+  if (!validSortQueries.includes(sort_by)) {
+    return Promise.reject({ status: 400, msg: "Invalid sort_by query" });
+  }
```

Comment:
The task wasn't asking for queries to be added - if you want to keep this code I would provide a test for it. The sort_by isn't being used in the query below either so this code isn't being used at the moment